src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const basePrice = days * 40;
+  const mediumTerm = 3;
+  const longTerm = 7;
+  const mediumTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  if (days < mediumTerm) {
+    return basePrice;
+  }
 
+  if (days >= mediumTerm && days < longTerm) {
+    return basePrice - mediumTermDiscount;
+  }
+
+  if (days >= longTerm) {
```

Comment:
You can remove this `if` statement because if the execution reaches this point, it means that `days` is necessarily greater than or equal to `longTerm`. Therefore, you can directly return the calculation without the `if` check.